.github/workflows/dt.yaml

```diff
@@ -0,0 +1,53 @@
+name: DefinitelyTyped
+on:
+  release:
+    types:
+      - created
```

Comment:
https://help.github.com/en/actions/reference/events-that-trigger-workflows#release-event-release